### ISSUE TITLE
Fire Elemental/Searing Totem Interaction

### ISSUE THREAD

**Issue by navv1234** [simulationcraft/simc#476]
Originally reported on Google Code with ID 477

```
What steps will reproduce the problem?
1. Simmed a roughly 350 ilevel ele shaman with different trinkets
2. Adjusted stats via reforging to make sure I was exactly hit capped then focused
on haste/mastery.  All gear except the trinkets were the exact same,
3. Compared Theralion's Mirror, Bell of Enraging Resonance, Soul Casket, and Stump
of Time

What is the expected output? What do you see instead?
My dps with Bell of Res is 20516, with Stump it is 20573, Theralion's Mirror is 20650,
Soul Casket is 20802.  

Based on my stat weights and napkin math, Stump should be the highest dps trinket with
Mirror and Casket being roughly equal.  Bell should be last.  So I am confused why
Stump and Bell are in last with nearly identical dps despite crit being terrible for
ele shaman and hit (which is largely reforged to haste/mastery while wearing the trinket).

What version of the product are you using? On what operating system?
Using 406-1 win 32 

Please provide any additional information below.

Basically, I am just trying to see if Stump of Time is being properly modeled for ele
shaman.
```

Reported by `christopher.cvijic` on 2011-02-16 05:57:00
